tests/Tests.AzureAppConfiguration/KeyVaultReferenceTests.cs

```diff
@@ -328,7 +328,7 @@ public void ThrowsWhenNoMatchingSecretClientIsFound()
         }
 
         [Fact]
-        public void ThrowsWhenSecretClientAndCredentialAreMissing()
+        public void ThrowsWhenSecretClientAndSecretResolverAndDefaultCredentialAreMissing()
```

Comment:
>ThrowsWhenSecretClientAndSecretResolverAndDefaultCredentialAreMissing [](start = 20, length = 69)

We could also use `ThrowsWhenConfigureKeyVaultIsMissing` or something similar, so we don't have to maintain it as we keep adding new parameters.